.circleci/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM ubuntu:18.04 as dev
+RUN apt-get update && apt-get install -y openjdk-11-jdk
+COPY --from=continuumio/miniconda3:4.8.2 /opt/conda /opt/conda
+ENV PATH="/opt/conda/bin/:${PATH}"
+ARG PYTORCH_CHANNEL=pytorch
+RUN conda install -c "${PYTORCH_CHANNEL}" -c powerai -y pytorch torchvision torchtext cpuonly psutil
```

Comment:
@seemethere Will this only run on CPU machines? How are we planning to generate binaries for GPUs?

Reply:
Does building with CUDA enabled binaries actually matter for using `torchserve` binaries with GPUs?

I'm unaware if it actually does.

Reply:
You are right, TorchServe binaries are not dependent on Cuda. We just need to verify that the generated binaries work for both CPU and GPU deployments